components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/filter/ExpressionNode.java

```diff
@@ -55,4 +56,12 @@ public void setAttributeValue(String attributeValue) {
 
         this.attributeValue = attributeValue;
     }
+
+    public String getSubAttributeValue() {
+        return subAttributeValue;
+    }
+
+    public void setSubAttributeValue(String subAttribute) {
+        this.subAttributeValue = subAttribute;
+    }
```

Comment:
missing new line after method signature

Reply:
This part is now removed after review changes.